client/domain/QuestionObjectFactorySpec.js

```diff
@@ -78,6 +85,10 @@ describe('QuestionObjectFactory', function() {
       expect(question.isLastTask(2)).toBe(false);
       expect(question.isLastTask(-1)).toBe(false);
       expect(question.isLastTask(1)).toBe(true);
+      expect(question.isLastTask(question.getTasks.length -
```

Comment:
Note that `question.getTasks` is a function, and does not have a length. (When you log `question.getTasks.length`, do you get a number?)

I'm also not sure what's going on here. Suppose `X is question.getTasks.length`. The argument here seems to be `X - (X - 1)`, or 1 -- which means that this is a more complicated way of writing the previous line of code?